src/data/dfinsupp.lean

```diff
@@ -721,6 +721,16 @@ lemma prod_neg_index [Π i, add_group (β i)] [Π i (x : β i), decidable (x ≠
   (-g).prod h = g.prod (λi b, h i (- b)) :=
 prod_map_range_index h0
 
+omit dec
+@[to_additive]
+lemma prod_comm {ι₁ ι₂ : Sort*} {β₁ : ι₁ → Type*} {β₂ : ι₂ → Type*}
+  [decidable_eq ι₁] [decidable_eq ι₂] [Π i, has_zero (β₁ i)] [Π i, has_zero (β₂ i)]
+  [Π i (x : β₁ i), decidable (x ≠ 0)] [Π i (x : β₂ i), decidable (x ≠ 0)] [comm_monoid γ]
+  (f₁ : Π₀ i, β₁ i) (f₂ : Π₀ i, β₂ i) (h : Π i, β₁ i → Π i, β₂ i → γ) :
+  f₁.prod (λ i₁ x₁, f₂.prod $ λ i₂ x₂, h i₁ x₁ i₂ x₂) =
+  f₂.prod (λ i₂ x₂, f₁.prod $ λ i₁ x₁, h i₁ x₁ i₂ x₂) := finset.prod_comm
+include dec
+
 omit dec
```

Comment:
```suggestion

```

Reply:
I did this because it's likely that lemmas might be inserted between, and this makes it harder to accidentally include the decidability. I suppose the linter would catch the mistake though, so it doesn't matter too much.